usr/src/cmd/rcm_daemon/common/rcm_event.c

```diff
@@ -121,8 +121,8 @@ process_event(int cmd, int seq_num, nvlist_t *nvl, nvlist_t **ret)
 	pid_t pid;
 	uint32_t flag = (uint32_t)0;
 	uint64_t pid64 = (uint64_t)0;
-	size_t buflen = 0;
-	size_t interval_size = 0;
+	uint_t buflen = 0;
```

Comment:
It feels like these being `size_t` was correct, and whatever used them was probably wrong, but I'm not sure.

Reply:
That would mean changing nvlist_lookup_byte_array() the fourth argument.

Reply:
Drat.  Ok, then yeah, this makes sense.